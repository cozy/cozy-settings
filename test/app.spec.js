'use strict'

/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

// import { I18nProvider } from 'cozy-ui/react/I18n'
// import App from '../src/components/App'

test('Hello world', () => {
  // const component = renderer.create(
  //   <I18nProvider lang='en'>
  //     <App />
  //   </I18nProvider>
  // )
  // const tree = component.toJSON()
  // expect(tree).toMatchSnapshot()
})
