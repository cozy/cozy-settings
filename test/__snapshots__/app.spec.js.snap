// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component only should be mounted correctly 1`] = `
<Layout
  monoColumn={false}
>
  <FlagSwitcher />
  <Alerter
    into="body"
  />
  <Sidebar />
  <Memo(RealTimeQueries)
    doctype="io.cozy.oauth.clients"
  />
  <Main>
    <Switch>
      <Route
        component={[Function]}
        path="/redirect"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/profile"
      />
      <Route
        component={[Function]}
        path="/profile/password"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/connectedDevices"
      />
      <Route
        component={[Function]}
        path="/connectedDevices/:deviceId"
      />
      <Route
        component={[Function]}
        path="/sessions"
      />
      <Route
        component={[Function]}
        path="/storage"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/permissions/:page"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/permissions/slug/:app"
      />
      <Route
        component={[Function]}
        path="/permissions/slug/:app/:permission"
      />
      <Route
        component={[Function]}
        path="/exports/:exportId"
      />
      <Redirect
        exact={true}
        from="/"
        push={false}
        to="/profile"
      />
      <Redirect
        exact={true}
        from="/permissions"
        push={false}
        to="/permissions/slug"
      />
      <Redirect
        from="*"
        push={false}
        to="/profile"
      />
    </Switch>
  </Main>
  <Sprite />
</Layout>
`;
